packages/connect/src/protocol-connect/handler-factory.ts

```diff
@@ -278,6 +280,8 @@ function createStreamHandler<I extends Message<I>, O extends Message<O>>(
     const outputIt = pipe(
       req.body,
       transformPrepend<Uint8Array>(() => {
+        if (opt.requireConnectProtocolHeader)
```

Comment:
probably just my OCD but I like curly braces or at least this to be on the same line.  I sorta wish we enforced https://eslint.org/docs/latest/rules/nonblock-statement-body-position

Reply:
:hear_no_evil: updated